irohad/ametsuchi/CMakeLists.txt

```diff
@@ -201,6 +214,7 @@ target_link_libraries(ametsuchi
     rdb_connection_init
     ametsuchi_rocksdb
     flat_file_storage
+    rocksdb_block_storage
```

Comment:
why flat file storege depends on rocksdb. It does not.

Reply:
???
This is dependencies for ```ametsuchi```.

Reply:
I thought ametsuchi_rocksdb should depend on rocksdb_block_storage

Reply:
```ametsuchi``` is not a flat file storage.